nvidia_nim/manifest.json

```
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "c7307eb9-7bbf-4dae-b74f-6396bf5bf514",
+  "app_id": "nvidia-nim",
+  "display_on_public_website": false,
```

```suggestion
  "display_on_public_website": false,
```
Just a heads up that this is set to false in case you want this to be published to the docs site.